enterprise/api_client/discovery.py

```diff
@@ -284,6 +284,22 @@ def get_program_course_keys(self, program_uuid):
             return []
         return [course['key'] for course in program_details.get('courses', [])]
 
+    def get_paginated_programs(self, querystring=None):
```

Comment:
Is this now unused btw?

Reply:
Yes. I will remove this and associated test code. Thanks.